internal/controller/accessrequest_controller.go

```diff
@@ -250,12 +295,25 @@ func (r *AccessRequestReconciler) getApplication(ctx context.Context, ar *api.Ac
 	return application, nil
 }
 
+func (r *AccessRequestReconciler) getRoleTemplate(ctx context.Context, ar *api.AccessRequest) (*api.RoleTemplate, error) {
+	roleTemplate := &api.RoleTemplate{}
+	objKey := client.ObjectKey{
+		Namespace: ar.Spec.Application.Namespace,
```

Comment:
I think the Role template should be located next to the target Application's project. Roles are more of a "AppProject" concern, rather than Application.

Reply:
Removed the reference from the application namespace and changed all AppProject related code to use the AccessRequest namespace. This means that the AppProject and the AccessRequest resources have to live in the same namespace (at least for now)